fuel-gson/build.gradle

```diff
@@ -0,0 +1,85 @@
+apply plugin: 'com.android.library'
+apply plugin: 'kotlin-android'
+
+apply plugin: 'com.novoda.bintray-release'
+
+apply plugin: 'jacoco-android'
+
+repositories {
+    jcenter()
+}
+
+android {
```

Comment:
What do you think about making this module Android-free? I think that Gson + Fuel could target to JVM itself.